test/java/seedu/addressbook/parser/NameTest.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.parser;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class NameTest {
+
+    @Test
+    public void varyingCapitalTest() throws IllegalValueException {
+        Name name1 = new Name("ABCDEFGH");
```

Comment:
going further you can consider moving all the steps necessary for the test to run (or preconditions) to a set up method before beginning the tests; correspondingly can put all the tear down in a method after the tests.